sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -770,7 +770,7 @@ int gnrc_netif_ipv6_group_idx(gnrc_netif_t *netif, const ipv6_addr_t *addr)
 }
 
 #if defined(MODULE_NETDEV_IEEE802154) || defined(MODULE_CC110X) || \
-    defined(MODULE_NRFMIN)
+    defined(MODULE_NRFMIN) || defined(MODULE_XBEE)
```

Comment:
The closing `#endif` does not get the `|| defined(MODULE_XBEE)` added  ([here](https://github.com/RIOT-OS/RIOT/pull/8490/files#diff-3eb20543397a7e6bb7064e71fac0471bR785))